R/unnormalize.R

```diff
@@ -17,10 +17,28 @@ unnormalize.default <- function(x, ...) {
 #' @export
 unnormalize.numeric <- function(x, verbose = TRUE, ...) {
   ## TODO implement algorithm include_bounds = FALSE
-  include_bounds <- attr(x, "include_bounds")
-  min_value <- attr(x, "min_value")
-  range_difference <- attr(x, "range_difference")
-  to_range <- attr(x, "to_range")
+
+  # if function called from the "grouped_df" method, we use the dw_transformer
+  # attributes that were recovered in the "grouped_df" method
+
+  dots <- match.call(expand.dots = FALSE)[["..."]]
+  grp_attr_dw <- eval(dots$grp_attr_dw, envir = parent.frame(1L))
+
+  if (!is.null(grp_attr_dw)) {
+    names(grp_attr_dw) <- gsub(".*\\.", "", names(grp_attr_dw))
```

Comment:
I can't say if this regex pattern is correct, but I guess it is and it's covered by tests?

Reply:
Thanks, I'll add some extra checks with other variables